rasa_core/actions/action.py

```diff
@@ -309,8 +310,13 @@ def run(self, dispatcher, tracker, domain):
                          "".format(self.name()))
             response = self.action_endpoint.request(
                     json=json, method="post", timeout=DEFAULT_REQUEST_TIMEOUT)
-            response.raise_for_status()
             response_data = response.json()
+            if response.status_code == 400:
+                logger.debug(response_data["error"])
+                return [ActionReverted(),
```

Comment:
I would let this action fail as well and let the processor handle this - if possible I'd rather have the tracker events look like this: `[..., ActionExecutionFailed, ...]` instead of `[..., ActionExecuted, ActionReverted, ActionExecutionFailed, ...]`